### PR TITLE
added is_valid_netmask +a test

### PR DESCRIPTION
**Pull request by pilferst**
# Description

- Added a function is_valid_netmask
- Added a test to test the function

### CODE REVIEW

**Comment by pilferst** [comment]
Added typedef mask. "netmask" is already used.

File: model/_init.cf [hunk 1]
```diff
@@ -37,6 +37,11 @@ typedef ip_v10 as string matching ip::is_valid_ip_v10(self) == true
     A string type that matches both v4 and v6 cidraddresses (hence the v10)
 """
 
+typedef mask as string matching ip::is_valid_netmask(self)  == true
```
Comment (bartv) [inline model/_init.cf:40]: In current versions of the language this should work as well.

```suggestion
typedef mask as string matching ip::is_valid_netmask(self)
```

**Review by bartv — CHANGES_REQUESTED** [review]
You need to fix the formatting as well using black:

https://jenkins.inmanta.com/job/modules/job/ip/job/issue%252Fis_valid_netmask/2/console

File: model/_init.cf (hunk 1 above)
Comment (pilferst) [reply model/_init.cf]: Fixed

**Review by bartv — APPROVED** [review]

**Comment by bartv** [comment]
@pilferst you should not merge it with the button, for module it is different

**Pull request merged**